packages/flutter_tools/lib/src/localizations/gen_l10n_types.dart

```diff
@@ -854,6 +854,7 @@ final Set<String> _iso639Languages = <String>{
   'lt',
   'lu',
   'lv',
+  'lzh',
```

Comment:
This list currently only contains ISO-639-1 language codes. We should probably investigate supporting all of ISO-639-2 or ISO-639-3 rather than adding patching this list with codes as users require them.

Reply:
Any thoughts?

Most language-data repositories still maintain their own language lists as far as I know.

Would adding language subtags that other projects already used/added/listed fit for you?

Reply:
I'm definitely open to this, I just need to consult some of my colleagues to figure out the right way forward to make sure we do this the right way. That might have to wait for next week as it's American Thanksgiving this week and a lot of the team will be out of office.

Reply:
Looks like this is the current language list Android using:

https://android.googlesource.com/platform/frameworks/base/+/648b4df/core/res/res/values/locale_config.xml